pyintegration/tests/test_table_listener.py

```diff
@@ -0,0 +1,106 @@
+#
+#   Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
+#
+import time
+import unittest
+
+from deephaven2 import time_table
+from deephaven2.table_listener import listen, TableListener
+
+from tests.testbase import BaseTestCase
+
+
+class TableListenerTestCase(BaseTestCase):
+    def setUp(self) -> None:
+        with self.ugp_lock_exclusive():
```

Comment:
Does this need a lock?

Reply:
yes, otherwise engine complains about no lock being held while attempting a table operation